book/error-monitoring/error-monitoring-ios.md

```diff
@@ -1,8 +1,9 @@
 
 # Error Monitoring plugin - iOS
+## Supported from iOS SDK 13 and above
```

Comment:
@kononenkoAnton I'm not sure it needs to be a subtitle, just a regular line can also do the work.